.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: node_js
-cache: yarn
 notifications:
   email: false
 node_js:
   - "4"
+  - "5"
```

Comment:
5 and 7 are useless dev releases and shouldn't be tested...

Reply:
Ah you mean only test the LTS versions. Yeah that makes sense.